logging/bin/create-sp-es-api-keys.sh

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env bash
+
+# create an Elastic Search API key for Service Provider
```

Comment:
service provider refers to third party node operators?

Reply:
Updated to use recommended verbiage.